src/data/store.js

```diff
@@ -303,6 +304,8 @@ const convertToServerRecord = (() => {
 
   function processTargetProps(context, source, target) {
     for (const prop in source) {
+      source[prop] = JSONUpdateBuilder.toUpdateObject(source[prop])
```

Comment:
see how it works with `Geometry` classes

Reply:
+